version.properties

```diff
@@ -1,4 +1,4 @@
-#Mon Sep 18 10:21:01 UTC 2023
-dependency.coreSdk.version=1.1.2
+#Mon Oct 02 12:10:17 UTC 2023
+dependency.coreSdk.version=1.1.4
```

Comment:
Why did the version jump?

Reply:
Because I forgot to do the postrelease action with 1.1.3